Makefile

```diff
@@ -89,3 +89,6 @@ uploadmessages:
 
 downloadmessages:
 	java -jar crowdin-cli.jar download -b `git symbolic-ref HEAD | xargs basename`
+
+distributefrontendmessages:
+	python ./kolibri/utils/distribute_frontend_messages.py
```

Comment:
Do you foresee this script being used after installation time? I'm not sure if we can totally eliminate the possiblity. Because if it won't be used, then we should move it outside of the `kolibri` module, so it doesn't get included in the sdist.

We should do that for all items that are only used during build time, and not needed during installation time, or at runtime.